.mergify.yml

```diff
@@ -11,6 +11,7 @@ pull_request_rules:
       - -title~=(WIP|wip)
       - -merged
       - -closed
+      - author!=dependabot[bot]
```

Comment:
Do we need [bot] in the author name?

Reply:
It tells mergify that the user type should be bot and not an actual user.